### ISSUE TITLE
Error on lerna run build in make file

### ISSUE THREAD

**Issue by chetan8300** [elyra-ai/elyra#634]
Platform: Windows 10
Yarn: 1.21.1
Node: 14.4.0
Python: 3.6.10
Jupyter Lab: 2.1.4
Elyra: 0.11.0.dev0

I am using conda enviroments to meet above software requirements and am having below errors on running `make clean install`.

```
export PATH=$(pwd)/node_modules/.bin:$PATH && lerna run build --npm-client=yarn
lerna notice cli v3.22.1
lerna info Executing command in 6 packages: "yarn run build"
lerna ERR! yarn run build exited 1 in '@elyra/ui-components'
lerna ERR! yarn run build stdout:
yarn run v1.21.1
$ tsc
../../node_modules/@blueprintjs/core/lib/esm/components/divider/divider.d.ts(258,28): error TS2707: Generic type 'UIEvent<T>' requires between 0 and 1 type arguments.
../../node_modules/@blueprintjs/core/lib/esm/components/divider/divider.d.ts(259,35): error TS2707: Generic type 'UIEvent<T>' requires between 0 and 1 type arguments.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:
error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in '@elyra/ui-components'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
make: *** [build-ui] Error 1
```

**Comment by kevin-bates** [comment]
Thanks for opening this issue.  It turns out to be platform-independent and is being addressed by PR #633.  As such, I'm going to remove [Windows] from the title.

It's great that you're looking into Windows builds as we have these other issues on Windows: #319 and #320

**Comment by vabarbosa** [comment]
Fixed by #633
